src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -91,20 +103,35 @@ static TenantAttributes tenantAttributes() {
   }
 
   static void postTenant() {
-    given().body(tenantAttributes()).
+    String id = given().body(tenantAttributes()).
     when().post("/_/tenant").
-    then().statusCode(201);
+    then().statusCode(201).
+    extract().
+    path("id");
+    await().until(() -> tenantFinishedLoading(id) == true);
```

Comment:
Does await have a default timeout in case the tenant initialisation never succeeds?

Reply:
I'm sure it has,. But why bother? await is the wrong solution to the problem.. It will fire HTTP GETs like mad.. We can just tell the async to return within 60 seconds or other.

Reply:
> We can just tell the async to return within 60 seconds or other.

Are you referring to the async within the RAML Module Builder implementation?

Reply:
No.

https://github.com/folio-org/raml/blob/d9ab6f377b24530ffbe92129caf63f5cb0814e7a/ramls/tenant.raml#L48

Reply:
Oh, the client can effectively make the (asynchronous by default) tenant API block till complete or a wait time has elapsed?

Reply:
yes. It as added so client did not have to poll too often (especially when we test).

Reply:
@adamdickmeiss Ok, thanks. What wait time would you advise?

Reply:
1 minute (60000) is fine. In some cases, it takes long time for Postgres to start.

Reply:
Ok. The client should wait for at least that long as well then.

@K-Felk please change this code to use the waiting variant of the tenant API as @adamdickmeiss advises with a timeout of 60 seconds / 1 minute.

Reply:
Yes.. I have posted the exact code to do that earlier for inspiration.

Reply:
Thanks @adamdickmeiss

Reply:
Changed.  Tests seem to be passing on my end, but they did before when using Await, so...